### PR TITLE
Add Slack invitation

### PR DESCRIPTION
**Pull request by egorikftp**

### CODE REVIEW

**Comment by Goooler** [comment]
This could be a badge like https://github.com/pinterest/ktlint/blob/48cc3f368248c837f81476b17384cb1ca75ed394/README.md?plain=1#L8.

**Comment by egorikftp** [comment]
Added badge

File: README.md [hunk 1]
```diff
@@ -231,6 +232,12 @@ other available gradle commands:
 
 - apply formatting: `./gradlew spotlessApply`
 
+## Join our community
+
+<a href="https://join.slack.com/t/composegears/shared_invite/zt-2noleve52-D~zrFPmC1cdhThsuQUW61A">
+    <img src="/assets/slack.png" alt="Slack" title="Slack" width="200">
+</a>
+
```
Comment (Goooler) [inline README.md:240]:
```suggestion
```

Comment (Goooler) [reply README.md:240]: I believe we can remove this now, just use the badge instead.

Comment (egorikftp) [reply README.md:240]: no, lets leave badge + section 🙂

File: assets/slack.png
Comment (Goooler) [inline assets/slack.png:1]: Replace this with a svg?

https://commons.wikimedia.org/wiki/File:Slack_Technologies_Logo.svg

**Review by Goooler — APPROVED** [review]

**Pull request merged**